app/views/layouts/application.html.erb

```diff
@@ -27,6 +27,7 @@
 
 <% content_for :body_end do %>
   <%= content_tag :div, nil, class: "js-AsyncGA", data: ga_tracking_data %>
+  <%= content_tag :div, nil, class: "js-Sentry", data: {sentry_js_dsn: config_item(:sentry_js_dsn)} %>
```

Comment:
Same as my other comment from pvb2, either remove the braces or add whitespace after / before the braces